src/goose.rs

```diff
@@ -811,21 +811,22 @@ impl GooseRequestCadence {
     }
 }
 
+pub trait GooseUserData: Downcast + Send + Sync + 'static {}
+impl_downcast!(GooseUserData);
+impl<T: Send + Sync + 'static> GooseUserData for T {}
+
 /// An individual user state, repeatedly running all [`GooseTask`](./struct.GooseTask.html)s
 /// in a specific [`GooseTaskSet`](./struct.GooseTaskSet.html).
-#[derive(Debug, Clone)]
```

Comment:
In your comments you mentioned removing `Clone`, but here you're also removing `Debug`. Either add it back here, or remove `Debug` from the various `GooseUserData` examples for consistency.

It's not strictly required here, but it can be quite useful so I'd lean toward adding it back unless there's an important reason to remove it.

Reply:
removed